selfdrive/car/hyundai/values.py

```diff
@@ -1105,6 +1107,31 @@ class Buttons:
       b'\xf1\x00bcsh8p54  U913\x00\x00\x00\x00\x00\x00TDL4T16NB05\x94t\x18',
     ],
   },
+  CAR.KIA_K5_HEV_2020: {
+    (Ecu.fwdRadar, 0x7D0, None): [
+      b'\xf1\x00DLhe SCC FHCUP      1.00 1.02 99110-L7000         ',
+    ],
+    (Ecu.eps, 0x7D4, None): [
+      b'\xf1\x00DL3 MDPS C 1.00 1.02 56310-L7000 4DLHC102',
+    ],
+    (Ecu.fwdCamera, 0x7C4, None): [
+      b'\xf1\x00DL3HMFC  AT KOR LHD 1.00 1.02 99210-L2000 200309',
+    ],
+    (Ecu.abs, 0x7D1, None): [
```

Comment:
can you provide routes for these versions?

Reply:
Yep! `c58dfc9fc16590e0|2023-01-14--13-51-48`

Reply:
Where did all the ABS and the other transmission come from?

Reply:
Oops, I think I accidentally copied it from another platform. The transmission FW is from `c58dfc9fc16590e0|2023-01-14--00-01-18--2`